app/scripts/modules/core/src/artifact/ArtifactIconService.ts

```diff
@@ -1,6 +1,20 @@
 import { ArtifactTypePatterns } from './ArtifactTypes';
 
-const unknownArtifactPath = require('./icons/unknown-type-artifact.svg');
+import unknownArtifactIcon from './icons/unknown-type-artifact.svg';
+import dockerIcon from './icons/docker-image-artifact.svg';
```

Comment:
side note: as an opponent of default imports, I might be inclined to add a linter rule that flags this line complaining that `dockerIcon` should be `dockerImageArtifact` or something similar.

Like? Hate?

Reply:
Yeah I like loosey goosey default imports less and less over time. Not against a lint rule. Would you think of the check as basically "imported name must equal file name transformed from kebab case to camel"?

I haven't ever written one myself, but given there's a bunch of examples where we're already looking at imports I'm guessing I could whip something up.